canonical-web-and-design/ubuntu.com
# Download promotions for server

## Done
- Added promotions in front of the server download

## QA
- Go to /download/server
- See that is matches [the copy doc](https://docs.google.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit)
- Check the steps work backwards and forwards
- Check the download works

Fixes https://github.com/canonical-web-and-design/web-squad/issues/3176
